deploy/e2e/apply/es6-master.yaml

```diff
@@ -37,43 +36,50 @@ spec:
           privileged: true
       containers:
       - name: elasticsearch
-        image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.6.1
-        env:
-        - name: ES_JAVA_OPTS
-          value: "-Xms256m -Xmx256m"
-        - name: node.master
-          value: "true"
-        - name: node.data
-          value: "false"
         resources:
           requests:
             memory: 512Mi
             cpu: 100m
           limits:
             memory: 512Mi
             cpu: 100m
-        ports:
-        - containerPort: 9300
-          name: transport
+        securityContext:
+          runAsUser: 1000
```

Comment:
Is there a reason we need to run as root?

Reply:
this is non-root, right? AFAIR Elasticsearch prefers to be run as non-root.

Reply:
sorry my mistake your right.